metrics/cli/docs/features/doctor_output_improvements/diagrams/ci_integrations_validator_class_diagram.puml

```diff
@@ -0,0 +1,114 @@
+@startuml ci_integrations_validator_class_diagram
+
+package core.lib.src.features.validation {
+    package model {
+        package builder {
+            class ValidationResultBuilder {
+                - _results: UnmodifiableMapView<ValidationTarget, TargetValidationResult>
+
+                + setResult(result: TargetValidationResult): void
+                + setEmptyResults(result: TargetValidationResult): void
+                + build(): ValidationResult
+            }
+        }
+
+        class ValidationTarget<T> {
+            + data: T
+            + name: String
+            + description: String
+        }
+
+        class ValidationConclusion<T> {
+            + data: T
```

Comment:
🔨
What do you think about removing the `data` field from this class? It seems like this value is never used and already contains in the `TargetValidationResult`.